org.gnome.Aisleriot.json

```diff
@@ -79,14 +79,13 @@
         },
         {
             "name": "aisleriot",
-            "config-opts": [
-                "--with-card-theme-formats=svg"
-            ],
```

Comment:
Needs an equivalent from meson iirc

Reply:
Well... checking https://gitlab.gnome.org/GNOME/aisleriot/-/blob/3.22.13/meson_options.txt it seems that is using svg-rsvg as default for `-Ddefault_theme_format`, and `-Dtheme_svg_rsvg` is true by default